packages/react-native/ReactCommon/jsc/JSCRuntime.cpp

```diff
@@ -36,6 +36,8 @@ class JSCRuntime : public jsi::Runtime {
  public:
   // Creates new context in new context group
   JSCRuntime();
+  // Creates new context in new context group with config
+  JSCRuntime(facebook::jsc::runtimeConfig rc);
```

Comment:
Should this be a reference?

Reply:
Yes, it is better to pass it as a reference: `const facebook::jsc::runtimeConfig&`.
Even if we have to copy the name, it is a very small cost.